website/docs/user-guides/analyze-workflow.md

```diff
@@ -4,4 +4,52 @@ title: Analyzing a Workflow
 sidebar_label: Analyzing Workflow
 ---
 
-## Coming soon
+---
+
+Workflow Analytics allows the users to analyze their workflows (Recurring/ non-Recurring) & helps them to keep track of the workflow runs of scheduled workflows and their behaviour.
```

Comment:
```suggestion
Workflow Analytics allows the users to analyze their workflows (Recurring/ Nonrecurring) & helps them to keep track of the workflow runs of scheduled workflows and their behaviour.
```
It'll be a single word https://www.dictionary.com/browse/nonrecurring#:~:text=adjective,to%20occur%20or%20happen%20again.
If used in other places we can change that.